### DISCUSSION TITLE
How to specify RTL-SDR dongle on configuration file

### DISCUSSION THREAD

**Discussion by 1beamer** [wmbusmeters/wmbusmeters#1172] [Q&A]
Hi, 

I am a newbie and maybe you can help me!
Can you tell me how to specify on wmbusmeters to use a RTL_SDR dongle instead of the generic  device=auto:t1 on configuration?
I am thinking that this will save some processing power& time eventually.

Other than this all if fine, the T1 telegram is correctly received and decoded.
On the documentation is mentioned but is not clear for me how to do this!

"Use auto for testing and to find your dongle. For production it is very much recommended that you change auto:t1 to the device name with the full device path (eg /dev/ttyUSB0:im871a:c1,t1). This will skip the slow probing for all possible wmbus dongles when wmbusmeters startup.

If the serial device (ttyUSB0) might change you can also use device=im871a:c1,t1 which will probe all serial devices but only scans for im871a which also speeds it up.

Note that the rtl-sdr devices are not found under the tty devices (e.g. /dev/tty...). Instead the rtl-sdr devices are accessed through character device special files named /dev/swradio0 to /dev/swradio255[1](https://github.com/wmbusmeters/wmbusmeters#user-content-fn-kernel_docs_sdr-54f7fa05b8a13b8625485a91e287c095). Wmbusmeters uses librtsldr to probe these devices."

**Comment by BIBOLV** [comment]
Hi,
Glad to hear that everything works for you and if it does not cause any issues you can leave `auto:t1`. Otherwise I would suggest to read trough documentation few more times carefully, as all details are there 😉 
But to answer your question - you can specify `device=rtlwmbus` if you have one dongle or `device=rtlwmbus[112233]` if you want specify specific dongle with specified serial. If you want to go even further and tune frequencies and other parameters - read on docs and issue archive, there is a lot of good information there.

**Reply by 1beamer** [reply]
Hi, thank you, I will do so!

**Reply by schildbach** [reply]
The first mentioned alternative (not specifying the ID) doesn't work. See this bug report: https://github.com/wmbusmeters/wmbusmeters/issues/1615
